src/less/lightbox-dialog/base/lightbox-dialog.less

```diff
@@ -0,0 +1,149 @@
+@import "../../mixins/utility/utility-mixins.less";
+
+.lightbox-dialog[role="dialog"] {
+    .dialog-base();
+}
+
+.lightbox-dialog__window {
+    .dialog-window();
+
+    flex-direction: column;
+    left: 0;
+    max-width: 616px;
```

Comment:
Any reason this was moved out of mixins. This is common across all dialogs except for mini
Maybe change it to a variable?

Reply:
Stylelint was complaining of duplicate max-width in the CSS. I think the mixins can be cleaned up and refactored in the next PR.